docker/Dockerfile-dev

```diff
@@ -4,7 +4,10 @@
 
 FROM python:3.5-alpine
 
+RUN apk --update --no-cache add \
+	sqlite > /dev/null 2>&1
 ADD requirements.txt /requirements.txt
+ADD docker/db/schema.db /sqlite.db
```

Comment:
Do we need to worry about database file permissions?  In prod the application server runs under a different user.